File: Demo/Objective_C_Demo/Cell/ChatTableViewCell.m
//
//  ChatTableViewCell.m
//  Demo
//
//  Created by IEMacBook01 on 22/05/16.
//  Copyright © 2016 Iftekhar. All rights reserved.
//

#import "ChatTableViewCell.h"

@implementation ChatLabel

-(CGSize)intrinsicContentSize
{
    CGSize sizeThatFits = [super intrinsicContentSize];
    sizeThatFits.width += 10;
    sizeThatFits.height += 10;
    return sizeThatFits;
}


@end

@implementation ChatTableViewCell

@end

File: Demo/Swift_Demo/Cell/ChatTableViewCell.swift
//
//  ChatTableViewCell.swift
//  Demo
//
//  Created by IEMacBook01 on 23/05/16.
//  Copyright © 2016 Iftekhar. All rights reserved.
//


class ChatLabel: UILabel {
    
    override var intrinsicContentSize : CGSize {

        var sizeThatFits = super.intrinsicContentSize
        sizeThatFits.width += 10
        sizeThatFits.height += 10

        return sizeThatFits
    }
}


class ChatTableViewCell: UITableViewCell {

    @IBOutlet var chatLabel : ChatLabel!
}
